tests/unit/lms/services/canvas_api/_basic_test.py

```diff
@@ -172,6 +186,10 @@ def paginated_results(self, http_session):
             factories.requests.Response(status_code=200),
         ]
 
+    @pytest.fixture
+    def paginated_results(self, http_session, paginated_responses):
```

Comment:
I think we normally name these side effect only fixtures something like 'with_paginated_results` to help distinguish them.